Templates.sln

```diff
@@ -230,6 +230,16 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ServiceBusTopicTrigger-Node
 		Templates\ServiceBusTopicTrigger-NodeJS\metadata.json = Templates\ServiceBusTopicTrigger-NodeJS\metadata.json
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TimerTrigger-Powershell", "TimerTrigger-Powershell", "{6B9E2925-C92E-43C2-96FE-8A6E0C5C7E52}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
Thanks for keeping the sln up to date - I'm having to go in and add things all the time that people missed :)